src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -628,7 +628,7 @@ await foreach (var item in enumerable)
         {
            return ReceiveWhileAsync(shouldContinue, max, idle, msgs, shouldIgnoreOtherMessageTypes, cancellationToken)
                 .ToListAsync(cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.